rts/lra/test/pom.xml

```diff
@@ -196,16 +221,109 @@
               </execution>
             </executions>
           </plugin>
+
+          <!-- Makes sure that the two Wildfly instances use different standalone XML configurations -->
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>${version.antrun.plugin}</version>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <phase>generate-test-resources</phase>
+                <configuration>
+                  <target>
+                    <copy file="${env.JBOSS_HOME}/standalone/configuration/standalone.xml" tofile="${env.JBOSS_HOME}/standalone/configuration/${lra.coordinator.xml.filename}"></copy>
+                    <copy file="${env.JBOSS_HOME}/standalone/configuration/standalone.xml" tofile="${env.JBOSS_HOME}/standalone/configuration/${lra.participant.xml.filename}"></copy>
+                  </target>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+
+          <!--
+            Wildfly is activated and configured through wildfly-maven-plugin instead
+            of using an Arquillian cluster. This design choice reflects the preference
+            to be consistent with the mechanism used to deploy lra-coordinator in
+            Quarkus. Moreover, the Microprofile LRA TCK does not allow to use multiple
+            Arquillian containers. As a consequence, lra-coordinator needs to be deployed
+            separately.
+          -->
+          <plugin>
+            <groupId>org.wildfly.plugins</groupId>
+            <artifactId>wildfly-maven-plugin</artifactId>
+            <version>${version.wildfly-maven-plugin}</version>
+            <configuration>
+              <skip>${skip.wildfly.plugin}</skip>
+            </configuration>
+
+            <executions>
+              <!-- Starts Wildfly -->
+              <execution>
+                <id>start</id>
+                <goals>
+                  <goal>start</goal>
+                  <goal>deploy-artifact</goal>
+                </goals>
+                <phase>pre-integration-test</phase>
+                <configuration>
+                  <!-- used by start -->
+                  <jbossHome>${env.JBOSS_HOME}</jbossHome>
+                  <serverConfig>${lra.coordinator.xml.filename}</serverConfig>
+
+                  <!-- used by deploy-artifact -->
+                  <groupId>org.jboss.narayana.rts</groupId>
+                  <artifactId>lra-coordinator-war</artifactId>
+                  <name>lra-coordinator.war</name>
```

Comment:
could be a property for easier changes later

Reply:
+1